sw/ground_segment/cockpit/live.ml

```diff
@@ -1521,6 +1532,45 @@ let get_shapes = fun (geomap:G.widget)_sender vs ->
 let listen_shapes = fun (geomap:G.widget) ->
   safe_bind "SHAPE" (get_shapes geomap)
 
+let get_cam_status = fun (geomap:MapCanvas.widget) _sender vs ->
```

Comment:
Isn't there some ambiguity if the camera is on a gimbal? It is unknown if the camera has any rotation in the direction it is pointing. Would it be better to move all the calculations into the server program and modify the cam status message to transmit the 4 corners of footprint. My current code also does not take into account any gimbal effects that the original may have, because the gimbal information is only being read on the server.